README.md

```diff
@@ -223,19 +193,19 @@ or run the helm install command with the `--set image.repository=YOUR_TAG_NAME`.
     sudo cp mc /usr/local/bin/mc
     ```
     Other OSes are detailed here https://docs.min.io/docs/minio-client-quickstart-guide.html
-    
-1. Publish the container definition for this project to a registry 
-   
+
+1. Publish the container definition for this project to a registry
+
    ```
    docker build -t REPOSITORYNAME:YOUR_TAG .
    docker push REPOSITORYNAME:YOUR_TAG
    ```
-   
+
 1. Modify the image definition in the yaml
 
     ```yaml
     image:
-    repository: REPOSITORYNAME:YOUR_TAG 
+    repository: REPOSITORYNAME:YOUR_TAG
```

Comment:
(nitpick and not related to the PR at all)

other charts conventionally try to split this up into three parts: registry, repository and tag. This was quite useful in helm, because you can override your registry per environment (imagine on-premise mirrors or cloud local repos). 
While testing the different tags with you, just updating a simple "tag" value instead of the whole string would've also been more convenient.

for example postgres:
https://artifacthub.io/packages/helm/bitnami/postgresql#postgresql-parameters